docs/index.md

```diff
@@ -50,7 +50,7 @@ This package will:
 2. Create another series of models that each represent one type of misalignment in your project (below you can find a full list of each misalignment and its accompanying model)
 3. Test those models to alert you to the presence of the misalignment
 
-Once you've installed the package, all you have to do is run a `dbt build --select package:dbt_project_evaluator`!
+Once you've installed the package, all you have to do is run a `dbt build --select package:dbt_project_evaluator`
```

Comment:
you removed my excitement !

Reply:
I  have heard a few times people trying to copy paste the excitement along with the command and then not getting what they want.

I guess dbt is not sentient *yet* and is unable to run commands in an excited way. Maybe one day! We can then re-add the `!`